anvil/src/udev.rs

```diff
@@ -58,20 +68,12 @@ use smithay::{
         seat::CursorImageStatus,
     },
 };
-#[cfg(feature = "egl")]
-use smithay::{
-    backend::{
-        drm::DevPath,
-        renderer::{ImportDma, ImportEgl},
-        udev::primary_gpu,
-    },
-    wayland::dmabuf::init_dmabuf_global,
-};
 
-use crate::{
-    drawing::*,
-    state::{AnvilState, Backend},
-};
+type UdevRenderer<'a> = MultiRenderer<'a, 'a, EglGlesBackend, EglGlesBackend, Gles2Renderbuffer>;
+#[cfg(feature = "debug")]
+smithay::custom_elements!(CustomElem; UdevRenderer<'_>; SurfaceTree=SurfaceTree, PointerElement=PointerElement::<MultiTexture>, FpsElement=FpsElement::<MultiTexture>);
+#[cfg(not(feature = "debug"))]
+smithay::custom_elements!(CustomElem; UdevRenderer<'_>; SurfaceTree=SurfaceTree, PointerElement=PointerElement::<MultiTexture>);
```

Comment:
Might be nicer to allow attaching a `meta` macro specifier to each variant of the enum generated like below:

```rust
smithay::custom_elements!(CustomElem; UdevRenderer<'_>;
    SurfaceTree=SurfaceTree,
    PointerElement=PointerElement::<MultiTexture>,
    #[cfg(feature = "debug")]
    FpsElement=FpsElement::<MultiTexture>
);
```